packages/protocol/scripts/sourcify-publish.ts

```diff
@@ -0,0 +1,92 @@
+/* tslint:disable no-console */
+import FormData from 'form-data'
+import fs from 'fs'
+import fetch from 'node-fetch'
+import path from 'path'
+import Web3 from 'web3'
+
+/*
+ * A script that reads the artifacts from the build/contracts directory and publish using the sourcify api.
+ *
+ * Expects the following flags:
+ *   network: The network for which artifacts should be
+ *
+ * Run using yarn run sourcify-publish, e.g.:
+ * yarn run sourcify-publish \
+ *   --network alfajores --buildArtifacts ./build/contracts --reportPath ./report.json
```

Comment:
By convention, we've been using `lowercase_with_underscores` for flags. Also, since we're using the upgrade proposal, not the version report, the last flag should probably be called `--proposal` instead.